### PR TITLE
Add retry button on the snackbar #107

### LINKED ISSUE #107: [Assigned] Add retry button on the snackbar
## Overview (Required)
https://github.com/DroidKaigi/conference-app-2018/pull/104#discussion_r161367945

## Links
-

### PR DESCRIPTION
**Pull request by Pluu**
## Issue
- close #107 

## Overview (Required)
- Refactor Extra Method, `SessionsViewModel#refreshSessions`
- Add, Snackbar Action
   - Snackback's Durataion was modified from Short to **Long**. Because I think the time is short until User recognizes Snackbar and pushes the button.
- Add, String Resource => R.string.session_load_retry

## Links
- None

## Screenshot
Before | After
:--: | :--:
 | 
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsViewModel.kt [hunk 1]
```diff
@@ -34,6 +34,10 @@ class SessionsViewModel @Inject constructor(
 
     @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
     fun onCreate() {
+        refreshSessions()
+    }
+
+    fun refreshSessions() {
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsViewModel.kt]: Please create `onRetrySessions()` ↩️ 
I want to separate intention and action. 

```
...
fun onCreate() {
       refreshSessions()
}
fun onRetrySessions() {
       refreshSessions()
}
private fun refreshSessions() {

```

Comment (Pluu) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsViewModel.kt]: No, Problem

**Comment by takahirom** [comment]
LGTM 👍

**Comment by Pluu** [comment]
@takahirom Thank you. I more learned in Review.

**Pull request merged**